plugins/TrollManagement/TrollManagementTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * @author David Barbier<david.barbier@vanillaforums.com>
+ * @copyright 2009-2021 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace VanillaTests\TrollManagement;
+
+use UserModel;
+use RoleModel;
+use VanillaTests\Models\UserModelTest;
+use VanillaTests\SetupTraitsTrait;
+
+/**
+ * Class TrollManagementTest
+ */
+class TrollManagementTest extends UserModelTest {
```

Comment:
Should really add some "banning" tests in here, since that's a large part of what this PR is introducing.

Reply:
Agreed.
One test has been added: a simple ban per fingerprint.
Trying to think of another plausible usage...